Installation issue: eigenexa

<!-- Thanks for taking the time to report this build failure. To proceed with the report please:

1. Title the issue "Installation issue: <name-of-the-package>".
2. Provide the information required below.

We encourage you to try, as much as possible, to reduce your problem to the minimal example that still reproduces the issue. That would help us a lot in fixing it quickly and effectively! -->

### Steps to reproduce the issue

<!-- Fill in the exact spec you are trying to build and the relevant part of the error message -->
```console
$ spack install eigenexa

$ spack install eigenexa
==> Warning: Missing a source id for fujitsu-mpi@3.0.0
[+] /vol0001/apps/oss/spack-v0.15.4/opt/spack/linux-rhel8-a64fx/fj-4.3.1/libsigsegv-2.12-2qhyocuw5rawo6tpqegh2xdaf623424q
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/berkeley-db-18.1.40-7dknf2wwpxx46qs4my6ahjivwozjzko3
[+] /vol0001/apps/oss/spack-v0.15.4/opt/spack/linux-rhel8-a64fx/fj-4.3.1/pkgconf-1.7.3-tkl6xodj7b7z63l7vmdakcknxo5ryend
==> fujitsu-mpi@3.0.0 : has external module in ['lang/tcsds-1.2.29']
[+] /opt/FJSVxtclanga/tcsds-1.2.29 (external fujitsu-mpi-3.0.0-nxj276sc2a4ppyq4wzew367ozc66agrh)
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/zlib-1.2.11-4cjuefhcsr5mnrfuxk7umqxhm5wwmtqf
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3
[+] /vol0001/apps/oss/spack-v0.15.4/opt/spack/linux-rhel8-a64fx/fj-4.3.1/m4-1.4.18-tajkcerbik3rbijuxty6rgp6dxk2qlvc
[+] /vol0001/apps/oss/spack-v0.15.4/opt/spack/linux-rhel8-a64fx/fj-4.3.1/ncurses-6.2-guk7ofvdpkdnrrhrannhuaji565i7buz
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/libtool-2.4.6-4iab5ptr3vmwfswxmz47irl7lgf35umt
[+] /vol0001/apps/oss/spack-v0.15.4/opt/spack/linux-rhel8-a64fx/fj-4.3.1/readline-8.0-n6urhogkjprjmdc7ivkh6bm7jb5xoi46
[+] /vol0001/apps/oss/spack-v0.15.4/opt/spack/linux-rhel8-a64fx/fj-4.3.1/gdbm-1.18.1-f5oceq4pkvekl6kaiplma2j75oiywr6q
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/perl-5.32.1-iami2xygpu2b4vbjzqqrlhkzzpnu33cz
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/autoconf-2.69-hwhwi3lfeoc4rg3ffcybtgrhnkredbew
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/openssl-1.1.1i-aybeapnv4ig4pw55ylgs6czo35sy4ygg
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/automake-1.16.3-k33afq3yuurzfhwu64a5cyyo6fag2yqe
[+] /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/cmake-3.19.2-jjurj4scdmtx2l6vhafuxwvl6hwr7gnu
==> Installing netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf
==> No binary for netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf found: installing from source
==> Fetching https://spack-llnl-mirror.s3-us-west-2.amazonaws.com/_source-cache/archive/61/61d9216cf81d246944720cfce96255878a3f85dec13b9351f1fa0fd6768220a6.tgz
######################################################################### 100.0%
==> netlib-scalapack: Executing phase: 'cmake'
==> netlib-scalapack: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

148 errors found in build log:
     5410    #define NDEBUG 1
     5411            ^
     5412    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
     5413    [ 70%] Building C object REDIST/TESTING/CMakeFiles/xdtrmr.dir/pdtr
             mrdrv.c.o
     5414    cd /vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-ne
             tlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-
             g6pr6ui/REDIST/TESTING && /opt/FJSVxtclanga/tcsds-1.2.29/bin/mpifc
             c -DAdd_ -I/opt/FJSVxtclanga/.common/MECA022/include/mpi/fujitsu -
             O3 -DNDEBUG -o CMakeFiles/xdtrmr.dir/pdtrmrdrv.c.o -c /data/hp2001
             39/u00444/u00444/spack-stage/spack-stage-netlib-scalapack-2.1.0-g6
             pr6uih4fbm5f36ql3beq2u4uuloplf/spack-src/REDIST/TESTING/pdtrmrdrv.
             c
     5415    [ 70%] Built target zpb2tim
  >> 5416    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_atan2'
  >> 5417    ../../lib/libscalapack.so: undefined reference to `__f_scmov'
  >> 5418    ../../lib/libscalapack.so: undefined reference to `__g_log10'
  >> 5419    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__f_aixi'
  >> 5420    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_datan2'
  >> 5421    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_csin'
     5422    [ 70%] Building C object PBLAS/TIMING/CMakeFiles/dpb2tim.dir/PB_Cw
             arn.c.o
     5423    /data/hp200139/u00444/u00444/spack-stage/spack-stage-netlib-scalap
             ack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-src/REDIST/TESTIN
             G/pdgemrdrv.c:475:7: warning: add explicit braces to avoid danglin
             g else [-Wdangling-else]
     5424          else
     5425          ^
  >> 5426    ../../lib/libscalapack.so: undefined reference to `__g_dlog'
     5427    cd /vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-ne
             tlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-
             g6pr6ui/PBLAS/TIMING && /opt/FJSVxtclanga/tcsds-1.2.29/bin/mpifcc
             -DAdd_ -DTestingPblas -I/opt/FJSVxtclanga/.common/MECA022/include/
             mpi/fujitsu -O3 -DNDEBUG -o CMakeFiles/dpb2tim.dir/PB_Cwarn.c.o -c
              /data/hp200139/u00444/u00444/spack-stage/spack-stage-netlib-scala
             pack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-src/PBLAS/TIMING
             /PB_Cwarn.c
  >> 5428    ../../lib/libscalapack.so: undefined reference to `__g_adxi'
  >> 5429    ../../lib/libscalapack.so: undefined reference to `__pkvla_cdexp'
  >> 5430    ../../lib/libscalapack.so: undefined reference to `__g_cexp'
  >> 5431    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_acxi'
  >> 5432    ../../lib/libscalapack.so: undefined reference to `__f_ccomp'
  >> 5433    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_axxi'
  >> 5434    ../../lib/libscalapack.so: undefined reference to `__g_arxr'
  >> 5435    ../../lib/libscalapack.so: undefined reference to `__g_dlog10'
  >> 5436    ../../lib/libscalapack.so: undefined reference to `__g_arxi'
  >> 5437    ../../lib/libscalapack.so: undefined reference to `__g_adxd'
  >> 5438    ../../lib/libscalapack.so: undefined reference to `__g_scns'
  >> 5439    ../../lib/libscalapack.so: undefined reference to `__jwe_isfm'
  >> 5440    ../../lib/libscalapack.so: undefined reference to `__g_csqrt'
  >> 5441    ../../lib/libscalapack.so: undefined reference to `__jwe_iocf'
  >> 5442    ../../lib/libscalapack.so: undefined reference to `__g_dexp'
  >> 5443    ../../lib/libscalapack.so: undefined reference to `__g_dcos'
  >> 5444    ../../lib/libscalapack.so: undefined reference to `__g_cdexp'
  >> 5445    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_cdsin'
  >> 5446    ../../lib/libscalapack.so: undefined reference to `__g_cdsqrt'
  >> 5447    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_sin'
  >> 5448    ../../lib/libscalapack.so: undefined reference to `__g_log'
  >> 5449    ../../lib/libscalapack.so: undefined reference to `__jwe_ilst'
  >> 5450    ../../lib/libscalapack.so: undefined reference to `__g_exp'
  >> 5451    ../../lib/libscalapack.so: undefined reference to `__g_dscn'
  >> 5452    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_dsin'
  >> 5453    ../../lib/libscalapack.so: undefined reference to `__pkvla_cexp'
  >> 5454    ../../lib/libscalapack.so: undefined reference to `__g_cos'
  >> 5455    clang-7: error: linker command failed with exit code 1 (use -v to
             see invocation)
  >> 5456    make[2]: *** [REDIST/TESTING/CMakeFiles/xitrmr.dir/build.make:108:
              scalapack/REDIST/TESTING/xitrmr] Error 1
     5457    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
  >> 5458    make[1]: *** [CMakeFiles/Makefile2:2199: REDIST/TESTING/CMakeFiles
             /xitrmr.dir/all] Error 2
     5459    make[1]: *** Waiting for unfinished jobs....
     5460    [ 70%] Building C object PBLAS/TIMING/CMakeFiles/dpb2tim.dir/PB_Ca
             bort.c.o
     5461    cd /vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-ne
             tlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-
             g6pr6ui/PBLAS/TIMING && /opt/FJSVxtclanga/tcsds-1.2.29/bin/mpifcc
             -DAdd_ -DTestingPblas -I/opt/FJSVxtclanga/.common/MECA022/include/
             mpi/fujitsu -O3 -DNDEBUG -o CMakeFiles/dpb2tim.dir/PB_Cabort.c.o -
             c /data/hp200139/u00444/u00444/spack-stage/spack-stage-netlib-scal
             apack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-src/PBLAS/TIMIN
             G/PB_Cabort.c
     5462    /data/hp200139/u00444/u00444/spack-stage/spack-stage-netlib-scalap
             ack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-src/REDIST/TESTIN
             G/pigemrdrv.c:154:9: warning: 'NDEBUG' macro redefined [-Wmacro-re
             defined]
     5463    #define NDEBUG
     5464            ^

     ...

     5514    cd /vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-ne
             tlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-
             g6pr6ui/REDIST/TESTING && /vol0004/hp200139/data/u00444/projects/s
             pack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/cmake-3.19.2-jjurj4scdmt
             x2l6vhafuxwvl6hwr7gnu/bin/cmake -E cmake_link_script CMakeFiles/xd
             trmr.dir/link.txt --verbose=1
     5515    Fortran diagnostic messages: program name(PB_DRAN)
     5516      jwd2008i-i  "/data/hp200139/u00444/u00444/spack-stage/spack-stag
             e-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-sr
             c/PBLAS/TIMING/pdblastim.f", line 2181: Dummy argument 'IDUMM' not
              used in this subprogram.
     5517    /opt/FJSVxtclanga/tcsds-1.2.29/bin/mpifcc -O3 -DNDEBUG -rdynamic C
             MakeFiles/xdtrmr.dir/pdtrmrdrv.c.o -o ../../scalapack/REDIST/TESTI
             NG/xdtrmr  -Wl,-rpath,/vol0004/hp200139/data/u00444/u00444/spack-s
             tage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4u
             uloplf/spack-build-g6pr6ui/lib:/vol0004/hp200139/data/u00444/proje
             cts/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/openblas-0.3.13-p25
             uejlm4zj6shfftneop7adgpbbvvh3/lib ../../lib/libscalapack.so /vol00
             04/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx
             /fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/lib/lib
             openblas.so
     5518    [ 70%] Linking Fortran executable dpb1tim
     5519    cd /vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-ne
             tlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-
             g6pr6ui/PBLAS/TIMING && /vol0004/hp200139/data/u00444/projects/spa
             ck/opt/spack/linux-rhel8-a64fx/fj-4.3.1/cmake-3.19.2-jjurj4scdmtx2
             l6vhafuxwvl6hwr7gnu/bin/cmake -E cmake_link_script CMakeFiles/dpb1
             tim.dir/link.txt --verbose=1
  >> 5520    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_atan2'
  >> 5521    ../../lib/libscalapack.so: undefined reference to `__f_scmov'
  >> 5522    ../../lib/libscalapack.so: undefined reference to `__g_log10'
  >> 5523    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__f_aixi'
  >> 5524    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_datan2'
  >> 5525    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_csin'
  >> 5526    ../../lib/libscalapack.so: undefined reference to `__g_dlog'
  >> 5527    ../../lib/libscalapack.so: undefined reference to `__g_adxi'
  >> 5528    ../../lib/libscalapack.so: undefined reference to `__pkvla_cdexp'
  >> 5529    ../../lib/libscalapack.so: undefined reference to `__g_cexp'
  >> 5530    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_acxi'
  >> 5531    ../../lib/libscalapack.so: undefined reference to `__f_ccomp'
  >> 5532    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_axxi'
  >> 5533    ../../lib/libscalapack.so: undefined reference to `__g_arxr'
  >> 5534    ../../lib/libscalapack.so: undefined reference to `__g_dlog10'
  >> 5535    ../../lib/libscalapack.so: undefined reference to `__g_arxi'
  >> 5536    ../../lib/libscalapack.so: undefined reference to `__g_adxd'
  >> 5537    ../../lib/libscalapack.so: undefined reference to `__g_scns'
  >> 5538    ../../lib/libscalapack.so: undefined reference to `__jwe_isfm'
     5539    /opt/FJSVxtclanga/tcsds-1.2.29/bin/mpifrt -rdynamic CMakeFiles/dpb
             1tim.dir/pdblas1tim.f.o CMakeFiles/dpb1tim.dir/pdblastim.f.o CMake
             Files/dpb1tim.dir/pblastim.f.o CMakeFiles/dpb1tim.dir/PB_Cwarn.c.o
              CMakeFiles/dpb1tim.dir/PB_Cabort.c.o -o dpb1tim   -L/opt/FJSVxtcl
             anga/tcsds-1.2.29/clang-comp/lib64  -L/usr/lib/gcc/aarch64-redhat-
             linux/8  -L/opt/FJSVxtclanga/tcsds-1.2.29/lib64  -L/opt/FJSVxos/mm
             m/lib64  -L/opt/FJSVxtclanga/tcsds-1.2.29/lib64/nofjobj  -Wl,-rpat
             h,/vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-net
             lib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-g
             6pr6ui/lib:/vol0004/hp200139/data/u00444/projects/spack/opt/spack/
             linux-rhel8-a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7a
             dgpbbvvh3/lib ../../lib/libscalapack.so /vol0004/hp200139/data/u00
             444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/openblas-0
             .3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/lib/libopenblas.so /vol0004
             /hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/f
             j-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/lib/libop
             enblas.so -lfjprofcore -lfjprofomp -lmpg -lm -lrt -lpthread -lelf
             -lz -ldl -lgcc -lgcc_s -lc -lgcc -lgcc_s
  >> 5540    ../../lib/libscalapack.so: undefined reference to `__g_csqrt'
  >> 5541    ../../lib/libscalapack.so: undefined reference to `__jwe_iocf'
  >> 5542    ../../lib/libscalapack.so: undefined reference to `__g_dexp'
  >> 5543    ../../lib/libscalapack.so: undefined reference to `__g_dcos'
  >> 5544    ../../lib/libscalapack.so: undefined reference to `__g_cdexp'
  >> 5545    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_cdsin'
  >> 5546    ../../lib/libscalapack.so: undefined reference to `__g_cdsqrt'
  >> 5547    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_sin'
  >> 5548    ../../lib/libscalapack.so: undefined reference to `__g_log'
  >> 5549    ../../lib/libscalapack.so: undefined reference to `__jwe_ilst'
  >> 5550    ../../lib/libscalapack.so: undefined reference to `__g_exp'
  >> 5551    ../../lib/libscalapack.so: undefined reference to `__g_dscn'
  >> 5552    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_dsin'
  >> 5553    ../../lib/libscalapack.so: undefined reference to `__pkvla_cexp'
  >> 5554    ../../lib/libscalapack.so: undefined reference to `__g_cos'
  >> 5555    clang-7: error: linker command failed with exit code 1 (use -v to
             see invocation)
  >> 5556    make[2]: *** [REDIST/TESTING/CMakeFiles/xdgemr.dir/build.make:108:
              scalapack/REDIST/TESTING/xdgemr] Error 1
     5557    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
  >> 5558    make[1]: *** [CMakeFiles/Makefile2:2226: REDIST/TESTING/CMakeFiles
             /xdgemr.dir/all] Error 2
     5559    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
     5560    [ 70%] Built target dpb3tim
     5561    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
     5562    [ 70%] Built target cpb3tim
  >> 5563    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_atan2'
  >> 5564    ../../lib/libscalapack.so: undefined reference to `__f_scmov'
  >> 5565    ../../lib/libscalapack.so: undefined reference to `__g_log10'
  >> 5566    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__f_aixi'
  >> 5567    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_datan2'
  >> 5568    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_csin'
  >> 5569    ../../lib/libscalapack.so: undefined reference to `__g_dlog'
  >> 5570    ../../lib/libscalapack.so: undefined reference to `__g_adxi'
  >> 5571    ../../lib/libscalapack.so: undefined reference to `__pkvla_cdexp'
  >> 5572    ../../lib/libscalapack.so: undefined reference to `__g_cexp'
  >> 5573    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_acxi'
  >> 5574    ../../lib/libscalapack.so: undefined reference to `__f_ccomp'
  >> 5575    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_axxi'
  >> 5576    ../../lib/libscalapack.so: undefined reference to `__g_arxr'
  >> 5577    ../../lib/libscalapack.so: undefined reference to `__g_dlog10'
  >> 5578    ../../lib/libscalapack.so: undefined reference to `__g_arxi'
  >> 5579    ../../lib/libscalapack.so: undefined reference to `__g_adxd'
  >> 5580    ../../lib/libscalapack.so: undefined reference to `__g_scns'
  >> 5581    ../../lib/libscalapack.so: undefined reference to `__jwe_isfm'
  >> 5582    ../../lib/libscalapack.so: undefined reference to `__g_csqrt'
  >> 5583    ../../lib/libscalapack.so: undefined reference to `__jwe_iocf'
  >> 5584    ../../lib/libscalapack.so: undefined reference to `__g_dexp'
  >> 5585    ../../lib/libscalapack.so: undefined reference to `__g_dcos'
  >> 5586    ../../lib/libscalapack.so: undefined reference to `__g_cdexp'
  >> 5587    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_cdsin'
  >> 5588    ../../lib/libscalapack.so: undefined reference to `__g_cdsqrt'
  >> 5589    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_sin'
  >> 5590    ../../lib/libscalapack.so: undefined reference to `__g_log'
  >> 5591    ../../lib/libscalapack.so: undefined reference to `__jwe_ilst'
  >> 5592    ../../lib/libscalapack.so: undefined reference to `__g_exp'
  >> 5593    ../../lib/libscalapack.so: undefined reference to `__g_dscn'
  >> 5594    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_dsin'
  >> 5595    ../../lib/libscalapack.so: undefined reference to `__pkvla_cexp'
  >> 5596    ../../lib/libscalapack.so: undefined reference to `__g_cos'
  >> 5597    clang-7: error: linker command failed with exit code 1 (use -v to
             see invocation)
  >> 5598    make[2]: *** [REDIST/TESTING/CMakeFiles/xigemr.dir/build.make:108:
              scalapack/REDIST/TESTING/xigemr] Error 1
     5599    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
  >> 5600    make[1]: *** [CMakeFiles/Makefile2:2253: REDIST/TESTING/CMakeFiles
             /xigemr.dir/all] Error 2
  >> 5601    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_atan2'
  >> 5602    ../../lib/libscalapack.so: undefined reference to `__f_scmov'
  >> 5603    ../../lib/libscalapack.so: undefined reference to `__g_log10'
  >> 5604    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__f_aixi'
  >> 5605    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_datan2'
  >> 5606    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_csin'
  >> 5607    ../../lib/libscalapack.so: undefined reference to `__g_dlog'
  >> 5608    ../../lib/libscalapack.so: undefined reference to `__g_adxi'
  >> 5609    ../../lib/libscalapack.so: undefined reference to `__pkvla_cdexp'
  >> 5610    ../../lib/libscalapack.so: undefined reference to `__g_cexp'
  >> 5611    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_acxi'
  >> 5612    ../../lib/libscalapack.so: undefined reference to `__f_ccomp'
  >> 5613    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_axxi'
  >> 5614    ../../lib/libscalapack.so: undefined reference to `__g_arxr'
  >> 5615    ../../lib/libscalapack.so: undefined reference to `__g_dlog10'
  >> 5616    ../../lib/libscalapack.so: undefined reference to `__g_arxi'
  >> 5617    ../../lib/libscalapack.so: undefined reference to `__g_adxd'
  >> 5618    ../../lib/libscalapack.so: undefined reference to `__g_scns'
  >> 5619    ../../lib/libscalapack.so: undefined reference to `__jwe_isfm'
  >> 5620    ../../lib/libscalapack.so: undefined reference to `__g_csqrt'
  >> 5621    ../../lib/libscalapack.so: undefined reference to `__jwe_iocf'
  >> 5622    ../../lib/libscalapack.so: undefined reference to `__g_dexp'
  >> 5623    ../../lib/libscalapack.so: undefined reference to `__g_dcos'
  >> 5624    ../../lib/libscalapack.so: undefined reference to `__g_cdexp'
  >> 5625    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_cdsin'
  >> 5626    ../../lib/libscalapack.so: undefined reference to `__g_cdsqrt'
  >> 5627    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_sin'
  >> 5628    ../../lib/libscalapack.so: undefined reference to `__g_log'
  >> 5629    ../../lib/libscalapack.so: undefined reference to `__jwe_ilst'
  >> 5630    ../../lib/libscalapack.so: undefined reference to `__g_exp'
  >> 5631    ../../lib/libscalapack.so: undefined reference to `__g_dscn'
  >> 5632    /vol0004/hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8
             -a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/l
             ib/libopenblas.so: undefined reference to `__g_dsin'
  >> 5633    ../../lib/libscalapack.so: undefined reference to `__pkvla_cexp'
  >> 5634    ../../lib/libscalapack.so: undefined reference to `__g_cos'
  >> 5635    clang-7: error: linker command failed with exit code 1 (use -v to
             see invocation)
  >> 5636    make[2]: *** [REDIST/TESTING/CMakeFiles/xdtrmr.dir/build.make:108:
              scalapack/REDIST/TESTING/xdtrmr] Error 1
     5637    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
  >> 5638    make[1]: *** [CMakeFiles/Makefile2:2280: REDIST/TESTING/CMakeFiles
             /xdtrmr.dir/all] Error 2
     5639    make[2]: Leaving directory '/vol0004/hp200139/data/u00444/u00444/s
             pack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3b
             eq2u4uuloplf/spack-build-g6pr6ui'
     5640    [ 70%] Built target dpb1tim
     5641    [ 70%] Linking Fortran executable dpb2tim
     5642    cd /vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-ne
             tlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-
             g6pr6ui/PBLAS/TIMING && /vol0004/hp200139/data/u00444/projects/spa
             ck/opt/spack/linux-rhel8-a64fx/fj-4.3.1/cmake-3.19.2-jjurj4scdmtx2
             l6vhafuxwvl6hwr7gnu/bin/cmake -E cmake_link_script CMakeFiles/dpb2
             tim.dir/link.txt --verbose=1
     5643    /opt/FJSVxtclanga/tcsds-1.2.29/bin/mpifrt -rdynamic CMakeFiles/dpb
             2tim.dir/pdblas2tim.f.o CMakeFiles/dpb2tim.dir/pdblastim.f.o CMake
             Files/dpb2tim.dir/pblastim.f.o CMakeFiles/dpb2tim.dir/PB_Cwarn.c.o
              CMakeFiles/dpb2tim.dir/PB_Cabort.c.o -o dpb2tim   -L/opt/FJSVxtcl
             anga/tcsds-1.2.29/clang-comp/lib64  -L/usr/lib/gcc/aarch64-redhat-
             linux/8  -L/opt/FJSVxtclanga/tcsds-1.2.29/lib64  -L/opt/FJSVxos/mm
             m/lib64  -L/opt/FJSVxtclanga/tcsds-1.2.29/lib64/nofjobj  -Wl,-rpat
             h,/vol0004/hp200139/data/u00444/u00444/spack-stage/spack-stage-net
             lib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-g
             6pr6ui/lib:/vol0004/hp200139/data/u00444/projects/spack/opt/spack/
             linux-rhel8-a64fx/fj-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7a
             dgpbbvvh3/lib ../../lib/libscalapack.so /vol0004/hp200139/data/u00
             444/projects/spack/opt/spack/linux-rhel8-a64fx/fj-4.3.1/openblas-0
             .3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/lib/libopenblas.so /vol0004
             /hp200139/data/u00444/projects/spack/opt/spack/linux-rhel8-a64fx/f
             j-4.3.1/openblas-0.3.13-p25uejlm4zj6shfftneop7adgpbbvvh3/lib/libop
             enblas.so -lfjprofcore -lfjprofomp -lmpg -lm -lrt -lpthread -lelf
             -lz -ldl -lgcc -lgcc_s -lc -lgcc -lgcc_s
     5644    Fortran diagnostic messages: program name(PILAENV)

See build log for details:
  /data/hp200139/u00444/u00444/spack-stage/spack-stage-netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf/spack-build-out.txt

==> Warning: Skipping build of eigenexa-2.6-4pjwuvqx7lbyaaqacxncaibekrehjkhi since netlib-scalapack-2.1.0-g6pr6uih4fbm5f36ql3beq2u4uuloplf failed
```

### Information on your system

<!-- Please include the output of `spack debug report` -->

<!-- If you have any relevant configuration detail (custom `packages.yaml` or `modules.yaml`, etc.) you can add that here as well. -->

### Additional information

<!-- Please upload the following files. They should be present in the stage directory of the failing build. Also upload any config.log or similar file if one exists. -->
* [spack-build-out.txt]()
* [spack-build-env.txt]()

<!-- Some packages have maintainers who have volunteered to debug build failures. Run `spack maintainers <name-of-the-package>` and @mention them here if they exist. -->

### General information

<!-- These boxes can be checked by replacing [ ] with [x] or by clicking them after submitting the issue. -->
- [ ] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [ ] I have run `spack maintainers <name-of-the-package>` and @mentioned any maintainers
- [ ] I have uploaded the build log and environment files
- [ ] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/openblas/package.py
+++ b/var/spack/repos/builtin/packages/openblas/package.py
@@ -277,6 +277,10 @@ class Openblas(MakefilePackage):
         if self.spec.satisfies('%gcc@:4.8.4'):
             make_defs.append('NO_AVX2=1')
 
+        # Fujitsu Compiler dose not add  Fortran runtime in rpath.
+        if self.spec.satisfies('%fj'):
+            make_defs.append('LDFLAGS=-lfj90i -lfj90f -lfjsrcinfo -lelf')
+
         # Newer versions of openblas will try to find ranlib in the compiler's
         # prefix, for instance, .../lib/spack/env/gcc/ranlib, which will fail.
         if self.spec.satisfies('@0.3.13:'):

